packages/runtime/src/runtime.ts

```diff
@@ -202,6 +202,17 @@ export class Runtime {
     return this.assertAccountDefined(address, account);
   }
 
+  /**
+   * Update sk for account.
+   * @param address account address
+   * @param sk secret key of account
+   */
+  updateAccountSecretKey (address: string, sk: Uint8Array): void {
+    const account = this.getAccount(address);
+    account.account.sk = sk;
+    this.store.accounts.set(address, account);
+  }
```

Comment:
why do we need this function? I don't think we should expose that in API.

Reply:
> why do we need this function? I don't think we should expose that in API.

If we don't provide this method. A user will be modify `runtime.store.accounts` Map directly. Is it OK?

Reply:
But where and why the user should modify the account's store key?

Reply:
Oh, I recheck In runtime we don't need sk to sign transaction. My bad. I think we don't want this method.

Reply:
yes, let's remove. I will pre-approve